Modix.Bot/Modules/UserInfoModule.cs

```diff
@@ -137,6 +150,44 @@ private async Task AddInfractionsToEmbed(ulong userId, StringBuilder builder)
             builder.AppendLine(FormatUtilities.FormatInfractionCounts(counts));
         }
 
+        private async Task AddParticipationToEmbed(ulong userId, StringBuilder builder)
+        {
+            var messages = await MessageRepository.GetRecentUserMessagesAsync(Context.Guild.Id, userId, TimeSpan.FromDays(30));
+
+            var lastWeek = DateTimeOffset.UtcNow - TimeSpan.FromDays(7);
+            var pastWeekCount = messages.Where(x => x.Timestamp >= lastWeek).Count();
```

Comment:
Minor, but I think `Count()` has an overload that takes a predicate

Reply:
Indeed it does, good idea. Updated.